Open Judge System/OJS.Services/Data/Problems/ProblemsDataService.cs

```diff
@@ -22,11 +22,6 @@ public class ProblemsDataService : IProblemsDataService
         public Problem GetWithContestById(int id) =>
             this.problems.All().Include(p => p.ProblemGroup.Contest).FirstOrDefault(p => p.Id == id);
```

Comment:
Why don't you move all method calls on separate lines?

Reply:
because they are less than 120 characters long